AndroidLibV2rayLite/interact.go

```diff
@@ -171,6 +160,17 @@ func (v *V2RayPoint) pointloop() error {
 	v.SupportSet.Prepare()
 	v.SupportSet.Setup(v.status.GetVPNSetupArg(v.EnableLocalDNS, v.ForwardIpv6))
 	v.SupportSet.OnEmitStatus(0, "Running")
+
+	if err := v.runTun2socks(); err != nil {
+		log.Println(err)
+		return err
+	}
+
+	log.Printf("EnableLocalDNS: %v\nForwardIpv6: %v\nDomainName: %s",
```

Comment:
Usually the log statement is at the beginning

Reply:
run tun2socks at the last.